### PR TITLE
mod : upkeep increase to go with reduced armor price

### PR DESCRIPTION
**Pull request by namidaka**

### CODE REVIEW

File: data/constants.json
Comment (verdie-g) [inline data/constants.json:1]: Please discard unrelated changes.

**Review by verdie-g — CHANGES_REQUESTED** [review]

File: data/constants.json
Comment (namidaka) [reply data/constants.json:1]: done

**Review by verdie-g — APPROVED** [review]
